src/Chart/Radar.php

```diff
@@ -547,16 +548,26 @@ public function drawRadar(Image $Object, Data $Values, array $Format = [])
                         $Points[$i][1],
                         $Points[$i + 1][0],
                         $Points[$i + 1][1],
-                        $Color
+                        $Color + (isset($Format['LineWeight']) ? [ 'Weight' => $Format['LineWeight'] ] : [] ),
                     );
                 }
                 if ($DrawPoints) {
-                    $Object->drawFilledCircle(
+                    $Object->drawShape(
                         $Points[$i][0],
                         $Points[$i][1],
+                        (isset($Series[$ID]['Shape']) ? $Series[$ID]['Shape'] : SERIE_SHAPE_CIRCLE),
                         $PointRadius,
-                        $Color
-                    );
+                        true,
+                        1,
+                        $Color['R'],
+                        $Color['G'],
+                        $Color['B'],
+                        $Color['Alpha'],
+                        0,
+                        0,
+                        0,
+                        100
+                    );    
```

Comment:
There seems to be a whitespace on this line.

Reply:
VScode keeps adding some here and there, I think I've found the extention that does it... Thanks!